### DISCUSSION TITLE
Setting up “Away” Automation

### DISCUSSION THREAD

**Discussion by mcooper333** [homebridge-simplisafe3/homebridge-simplisafe3#252] [Q&A] [answered]
Thanks for keeping this great! Been trying to creat an automation for when SimpliSafe is “Away,” it would trigger my thermostats to away. Unfortunately, the only options are triggered, armed and disarmed. My concern is that when I set the SimpliSafe into “Home” mode, it would still trigger the automation. Are there any workarounds for this? I noticed in shortcuts, it gives me the away and home option but I believe the is only triggered with Siri. Any thoughts would be much appreciated - thanks! 

![CF67A53D-9C80-4C37-9F94-608436E989CB](https://user-images.githubusercontent.com/92616054/138367892-9407389d-140f-4a08-a326-d8d542639d7e.png)

**Comment by shamoon** [comment, marked as answer]
Hey @mcooper333, a couple thoughts:

1. What about just using a completely separate automation for the thermostat? Like "when the last person leaves" --> "set thermostat to ___"? Thats what I do for my alarm as well (actually for the alarm you have to use a 'dummy switch' to make this totally automatic but hopefully you get the idea).
2. Try editing the automation in the (free) Eve app which often gives more options and I think includes options for ~home arm

Ultimately this plugin doesnt have control over which choices HomeKit shows but again, I think theyre there which is why the Eve app will likely do the trick.

**Reply by mcooper333** [reply]
Thanks for the feedback! Whole idea to connect the thermostat to simplisafe is that it ensures no one is home when the thermostat is set to away. Geofence option is good but we have a kid. So if they are left with a baby sitter with both my wife and I not in the house, we'd have to remember to make sure the thermostats are still set to "Home" mode. 

I'll give the eve app a go! Thank you again!

**Reply by shamoon** [reply]
Cool. 

Btw, I have a similar thing with young children and I recently implemented a “babysitter switch” using homebridge-dummy. It turns on for 15 minutes and I have my “Leave Home” automation (which uses geofencing) set to _not_ execute if the babysitter switch is on. So at some point before we leave I just have to remember to ask Siri to “turn on the babysitter switch” and voila! :)

**Reply by mcooper333** [reply]
This is a great idea! I also just confirmed that the eve app does what I want it to do so thank you for that suggestion. 

Do you have any familiarity with occupancy sensors? I had a lutron cadets sensor that worked great but I’m not in a new house with a different set up. I feel like there are only motion sensors for homekit which defeats the purpose if I’m sitting in my office.

**Reply by shamoon** [reply]
Cool! 

No sorry don’t know about occupancy sensors, there are some helpful Reddit groups you might want to look into https://reddit.com/r/HomeKit and https://reddit.com/r/homebridge
